nucypher/blockchain/eth/signers.py

```diff
@@ -188,8 +193,15 @@ def sign_transaction(self, transaction_dict: dict) -> HexBytes:
             'chainId': Web3.toHex,
             'from': to_checksum_address
         }
-        transaction_dict = apply_formatters_to_dict(formatters, transaction_dict)
-        signed = self.__ipc_request("account_signTransaction", transaction_dict)
+
+        # Workaround for contract creation TXs
```

Comment:
This pattern appears several times around the codebase - Do you think this transaction dict mutation is appropriately handled at this abstraction layer or a higher one?